app/src/js/Draw.js

```diff
@@ -0,0 +1,91 @@
+import "leaflet";
+import "leaflet-draw";
+import Wkt from "wicket";
+import { func } from "prop-types";
+
+
+export default L.Control.AstroDraw = L.Control.Draw.extend({
+	options: {
+		position: 'topleft',
+		draw: {},
+    edit: false
+  },
+
+  onAdd: function (map) {
+    let container = L.DomUtil.create('div', 'leaflet-draw'),
+			addedTopClass = false,
+			topClassName = 'leaflet-draw-toolbar-top',
+      toolbarContainer;
+
+		for (var toolbarId in this._toolbars) {
+			if (this._toolbars.hasOwnProperty(toolbarId)) {
+				toolbarContainer = this._toolbars[toolbarId].addToolbar(map);
+
+				if (toolbarContainer) {
+					if (!addedTopClass) {
+						if (!L.DomUtil.hasClass(toolbarContainer, topClassName)) {
+							L.DomUtil.addClass(toolbarContainer.childNodes[0], topClassName);
+						}
+						addedTopClass = true;
+					}
+
+					container.appendChild(toolbarContainer);
+				}
+			}
+    }
+
+     this.myLayer = L.geoJSON().addTo(map);
+
+    this.wktButton = L.DomUtil.get("wktButton");
+    L.DomEvent.on(this.wktButton, "click", this.mapWKTString, this);
+
+
+    map.on("draw:created", this.shapesToWKT, this);
+
+    return container;
+  },
+
+  shapesToWKT: function(e){
+    this.myLayer.clearLayers();
+    this.options.edit['featureGroup'].clearLayers();
+    let wktTextBox = L.DomUtil.get("wktTextBox");
+    let wkt = new Wkt.Wkt();
```

Comment:
Can we move this to another method so that every time a shape is drawn we don't have to recreate this variables?